core/sql/optimizer/RelExpr.cpp

```diff
@@ -9515,7 +9515,8 @@ FileScan::FileScan(const CorrName& tableName,
      estRowsAccessed_ (0),
      mdamFlag_(UNDECIDED),
      skipRowsToPreventHalloween_(FALSE),
-     doUseSearchKey_(TRUE)
+     doUseSearchKey_(TRUE),
+     computedNumOfActivePartiions_(-1)
```

Comment:
Will do. Sorry about it.

On Mon, Oct 19, 2015 at 8:29 PM, Hans Zeller <notifications@github.com>
wrote:

> In core/sql/optimizer/RelExpr.cpp
> <https://github.com/apache/incubator-trafodion/pull/132#discussion_r42446838>
> :
>
> > @@ -9515,7 +9515,8 @@ FileScan::FileScan(const CorrName& tableName,
> >       estRowsAccessed_ (0),
> >       mdamFlag_(UNDECIDED),
> >       skipRowsToPreventHalloween_(FALSE),
> > -     doUseSearchKey_(TRUE)
> > +     doUseSearchKey_(TRUE),
> > +     computedNumOfActivePartiions_(-1)
>
> Partitions is misspelled 8 times in this file, might make sense to fix the
> spelling for all of those.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/132/files#r42446838>.
>



-- 
Regards, --Qifan